ui/src/views/repositories/index.tsx

```diff
@@ -1,33 +1,49 @@
-import React from 'react'
+import React, { useEffect } from 'react'
 import { useRepositoriesContext, useRepositoriesSetState } from 'src/state/contexts/repositories.context'
 import { EmptyRepositoryTable, FilterHeader, PageHeader, RepositoriesTable } from './components'
 import { SyncDataDropDown } from './drop-downs/sync-repos-data-drop-down'
 import { AddRepositoryModal } from './modals/add-repository-modal'
 import { ManageAutoImportReposModal } from './modals/auto-import-repository-modals/manage-auto-imports-modal'
 import { SyncAutoImportReposModal } from './modals/auto-import-repository-modals/sync-auto-import-modal'
 
-import { Button, Toolbar } from '@mergestat/blocks'
+import { Alert, Button, Toolbar } from '@mergestat/blocks'
 import { CaretDownIcon, PencilIcon, RefreshIcon } from '@mergestat/icons'
 import Loading from 'src/components/Loading'
+import { useGeneralContext, useGeneralSetState } from 'src/state/contexts'
 import useRepos from './hooks/useRepos'
 
 const RepositoriesView: React.FC = () => {
   const [{ showAddRepositoryModal, showAutoImportModal, showSyncRepoModal, search }] = useRepositoriesContext()
+  const [{ autoImportingRepos }] = useGeneralContext()
 
   const { setShowAutoImportModal, setShowSyncRepoModal } = useRepositoriesSetState()
+  const { setAutoImportingRepos } = useGeneralSetState()
 
   const { showTable, loading, data } = useRepos(search)
 
   // - Todo: connect selectedRepositoriesCount from RepositoriesTable
   const selectedRepositoriesCount = 0
 
+  useEffect(() => {
+    autoImportingRepos && setTimeout(() => {
+      setAutoImportingRepos(false)
+    }, 20000)
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [autoImportingRepos])
+
   return (
     <main className="w-full flex flex-col h-full bg-gray-50 overflow-hidden">
       <div className="bg-white border-b border-gray-200">
         <PageHeader />
         {showTable && <FilterHeader />}
       </div>
       <div className="flex-1 items-center p-8 overflow-auto">
+        {autoImportingRepos &&
+          <Alert type="info" theme="light" title="Auto importing repos" className='mb-5'>
+            The repos from the auto import setting will appear shortly once synced.
+          </Alert>
+        }
```

Comment:
instead of this appearing only when the user creates a new auto-import, can we have it appear based on the result of a graphql query? We want it to show up when an auto-import _exists_ in the database, but the `last_import` timestamp is `NULL`. Does that make sense? That way this will persist across page refreshes as well

Reply:
Done [fbbab8e](https://github.com/mergestat/fuse/pull/187/commits/fbbab8e107f753adfa46f0242be0c47ef8aeef57)